.claude/skills/e2e-tests.md

```
@@ -0,0 +1,60 @@
+name: e2e-tests
+description: Run e2e tests, auto-detecting hostmode from the cluster and supporting test filtering
+trigger: run e2e tests, run tests, e2e, run the e2e suite, test e2e
+---
+
+# Run e2e tests
+
+## Step 1: Detect if the cluster runs in hostmode (systemd mode)
+
+Use the kubeconfig at `bin/kubeconfig` to check for hostbridge pods:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)